raft/src/test/java/org/apache/kafka/raft/RaftClientTestContext.java

```diff
@@ -625,10 +626,10 @@ void deliverResponse(int correlationId, Node source, ApiMessage response) {
         channel.mockReceive(new RaftResponse.Inbound(correlationId, response, source));
     }
 
-    RaftRequest.Outbound assertSentBeginQuorumEpochRequest(int epoch, int numBeginEpochRequests) {
+    void assertSentBeginQuorumEpochRequest(int epoch, int numBeginEpochRequests, Set<Integer> destinationIds) {
```

Comment:
`numBeginEpochRequests` is implicit from the size of the `destinationIds` set. In other words, `assertEquals(numBeginEpochRequests, destinationIds)` is always true so you can remove the parameter and replace `numBeginEpochRequests` in this method with `destinationIds.size()`.

Reply:
I was thinking this would also check that we don't send more than one request to a destination